ui/src/components/credentials/IssuedQR.tsx

```diff
@@ -0,0 +1,110 @@
+import { Card, Col, Grid, Image, Row, Space, Typography } from "antd";
+import { QRCodeSVG } from "qrcode.react";
+import { useCallback, useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+
+import { APIError } from "src/adapters/api";
+import { getIssuedQRCode } from "src/adapters/api/credentials";
+import { ErrorResult } from "src/components/shared/ErrorResult";
+import { LoadingResult } from "src/components/shared/LoadingResult";
+import { useEnvContext } from "src/contexts/Env";
+import { AsyncTask, hasAsyncTaskFailed, isAsyncTaskDataAvailable } from "src/utils/async";
+import { isAbortedError, makeRequestAbortable } from "src/utils/browser";
+import { WALLET_APP_STORE_URL, WALLET_PLAY_STORE_URL } from "src/utils/constants";
+
+export function IssuedQR() {
+  const env = useEnvContext();
+  const [issuedQRCode, setIssuedQRCode] = useState<AsyncTask<unknown, APIError>>({
+    status: "pending",
+  });
+
+  const { lg } = Grid.useBreakpoint();
+  const { credentialID } = useParams();
+
+  const createCredentialQR = useCallback(
+    async (signal: AbortSignal) => {
+      if (credentialID) {
+        setIssuedQRCode({ status: "loading" });
+
+        const response = await getIssuedQRCode({ credentialID, env, signal });
+
+        if (response.isSuccessful) {
+          setIssuedQRCode({ data: response.data, status: "successful" });
+        } else {
+          if (!isAbortedError(response.error)) {
+            setIssuedQRCode({ error: response.error, status: "failed" });
+          }
+        }
+      }
+    },
+    [credentialID, env]
+  );
+
+  useEffect(() => {
+    const { aborter } = makeRequestAbortable(createCredentialQR);
+
+    return aborter;
+  }, [createCredentialQR]);
+
+  const onStartAgain = () => {
+    makeRequestAbortable(createCredentialQR);
+    setIssuedQRCode({ status: "pending" });
+  };
+
+  const hasFailed = hasAsyncTaskFailed(issuedQRCode) ? issuedQRCode.error : undefined;
+
+  if (hasFailed) {
+    return (
+      <ErrorResult error={hasFailed.message} labelRetry="Start again" onRetry={onStartAgain} />
+    );
+  }
+
+  if (!isAsyncTaskDataAvailable(issuedQRCode)) {
+    return <LoadingResult />;
+  }
+
+  return (
+    <Space align="center" direction="vertical" size="large">
```

Comment:
Let's abstract the reusable DOM between this and `CredentialLinkQR`.

Reply:
I don't understand what you want to reuse, let's see if this page persists in the future and we'll think about it

Reply:
I'd not abstract anything here. There's little repetition and as Adri points out we don't know for how long this page will make it.

Reply:
Proposal for `CredentialQR` abstraction: https://github.com/0xPolygonID/sh-id-platform/pull/366